packages/terra-consumer-layout/src/Layout.jsx

```diff
@@ -68,6 +77,17 @@ class Layout extends React.Component {
             />
           </nav>
           <main id="main-container" className={cx('main-container')}>
+            {siteAlert.map(
+              alert => (
+                <Alert
+                  key={`alert-${alert.alertId}`}
+                  type={Alert.Opts.Types.WARNING}
```

Comment:
do we want to assume all site alerts are warnings? I don't think we do. This is another reason I believe the alerts should literally be an array of TerraAlert component.

Reply:
Yes all the site alerts are warnings.I guess that's the requirement for healthe-elastic.

Reply:
Should I declare all alert types in props and pass the alert type specific to that alert banner with the contents of banner?

Reply:
Why not just push that deciion out of this UI compoennt and into the app framework. Just have this take an array (or single) alert component. I don't understand the benefit of passing in the data state like this

Reply:
Yes ,you are right. I will let the app framework decide the alert type.